qbit_manage.py

```diff
@@ -497,6 +498,10 @@ def finished_run():
         if cfg.commands["tag_update"]:
             stats["tagged"] += Tags(qbit_manager).stats
 
+        # Remove cross-seed tags
```

Comment:
shouldn't this be done after the new cross-seed tags are added?

Reply:
Yes it would remove the tags thereafter